### ISSUE TITLE
Peer review

### ISSUE THREAD

**Issue by ZhifeiJin** [yashmundra13/ORIE4741#2]
**Summary**
This project is about prediction of water contamination using data on hydraulic fracturing sites. They will be using data from EPA and FracFocus, which seem to be valid and specific dataset that will be useful to find the answer to the question.
**What I like about the project**
I think it is a very interesting project and I definitely learned a lot just by reading your proposal. You have a very specific question outlined, and it definitely seems feasible. You also have promising data and you clearly have thought a lot about what you will have do with it, and I think that is awesome. 
**What I think could be improved**
I'm not sure what are all of the features that you will be using, but I'm wondering if there will be other confounding variables. For example, I think the local facilities (such as factories) might also influence the water contamination. Also, you mentioned that you will limit your data to several states to ensure your effectiveness of data analysis. I think a) it might not count as a big messy data (without knowing how large the dataset will be), and b) you will need to validate your result using  a test set to show that your model does not overfit to the training set.
